packages/cache-persist/src/createStorage.native.ts

```diff
@@ -1,20 +1,5 @@
-/* eslint import/no-extraneous-dependencies:0 */
```

Comment:
I think this is necessary 👍

Reply:
On that note, is there a reason this is only listed as a development dependency?